project_pivot/readme/ROADMAP.rst

```diff
@@ -1 +1 @@
-This module is not compatible with 'project_list'
+This module is not compatible with 'project_list' and 'project_timeline'
```

Comment:
Could you document the reason.
```suggestion
This module is not compatible with 'project_list' and 'project_timeline' because both modules replace the same actions
```

Reply:
Done